en/config_rover/ackermann.md

```diff
@@ -30,70 +28,224 @@ To configure the Ackermann rover frame and outputs:
-Therefore these two parameters have to be set for the slew rates to work!
+5. [RA_MAX_DECEL](#RA_MAX_DECEL) [m/s^2]: Maximum deceleration you want to allow for your rover.
+
+   :::tip
+   The same [considerations](#RA_MAX_ACCEL_CONSIDERATIONS) as in the configuration of [RA_MAX_ACCEL](#RA_MAX_ACCEL) apply.
+   :::
+
+   :::info
+   This parameter is also used for the calculation of the speed setpoint during [Auto modes](#auto-modes).
+   :::
+
+6. (Optional) [RA_MAX_STR_RATE](#RA_MAX_STR_RATE) [deg/s]: Maximum steering rate you want to allow for your rover.
+
+   :::tip
+   This value is set to your preference, there is no general rule of thumb since it depends entirely on your rover and use case.
+   For bigger rovers there might be a mechanical limit that is easy to identify by steering the rover at a standstill and increasing
+   [RA_MAX_STR_RATE](#RA_MAX_STR_RATE) until you observe the steering rate to be limited by the parameter.
+   For smaller rovers you might observe the steering to be too aggressive. Set [RA_MAX_STR_RATE](#RA_MAX_STR_RATE) to a low value and steer the rover at a standstill.
+   Increase the parameter until you reach the maximum steering rate you are comfortable with.
+   :::
+
+   :::warning
+   A low maximum steering rate makes the rover worse at tracking steering setpoints, which can lead to a poor performance in the subsequent modes.
+   :::
+
+## Acro Mode
+
+::: warning
+For this mode to work properly [Manual mode](#acro-mode) must've already been configured!
 :::
 
-## Mission Parameters
+To set up [Acro mode](../flight_modes_rover/ackermann.md#acro-mode) configure the following [parameters](../advanced_config/parameters.md) in QGroundControl:
+
+1. [RA_MAX_LAT_ACCEL](#RA_MAX_LAT_ACCEL): Maximum lateral acceleration you want to allow for your rover.
+
+   :::tip
+   Limiting the lateral acceleration is necessary if the rover is prone rolling over, loosing traction at high speeds or if passenger comfort is important.
+   Small rovers especially can be prone to rolling over when steering aggressively at high speeds.
+
+   If this is the case:
 
-These parameters only affect vehicle in [Mission Mode](../flight_modes_rover/ackermann.md#mission-mode).
+   1. In [Acro mode](../flight_modes_rover/ackermann.md#acro-mode), set [RA_MAX_LAT_ACCEL](#RA_MAX_LAT_ACCEL) to a small value and drive the rover at full throttle and with the right stick all the way to the left or right.
+   1. Increase [RA_MAX_LAT_ACCEL](#RA_MAX_LAT_ACCEL) until the wheels of the rover start to lift up.
+   1. Set [RA_MAX_LAT_ACCEL](#RA_MAX_LAT_ACCEL) to the highest value that does not cause the rover to lift up.
+
+   The same approach of slowly increasing [RA_MAX_LAT_ACCEL](#RA_MAX_LAT_ACCEL) from a small value can be taken if you encounter any of the other aforementioned issues.
```

Comment:
I don't understand what you are saying in this line. What issues - we've just tuned this.

Reply:
This was in referene to

> Limiting the lateral acceleration is necessary if the rover is prone rolling over, loosing traction at high speeds or if passenger comfort is important.

But I agree that it might be confusing I changed the wording [here](https://github.com/PX4/PX4-user_guide/pull/3407/commits/32ad89974d3157051edd5a79d345ff84fd5226f4).

Reply:
Yes, that is now clear.